src/bma250e/javaupm_bma250e.i

```diff
@@ -1,10 +1,25 @@
 %module javaupm_bma250e
 %include "../upm.i"
 %include "typemaps.i"
-%include "../upm_vectortypes.i"
+%include "../upm_stdvector.i"
 
 %ignore getAccelerometer(float *, float *, float *);
 
+%typemap(javaimports) SWIGTYPE %{
+import java.util.AbstractList;
+import java.lang.Float;
+%}
+
+%typemap(javaout) upm::BMA250E {
+    return new $&javaclassname($jnicall, true);
+}
+%typemap(javaout) std::vector<float> {
+    return (AbstractList<Float>)(new $&javaclassname($jnicall, true));
+}
+%typemap(jstype) std::vector<float> "AbstractList<Float>"
+
+%template(floatVector) std::vector<float>;
+
```

Comment:
Hi, is this stanze only needed for the Float types in Java?  I notice you did not add these for Int's and uint8_t's (chars/bytes).

This stanza also appears to be duplicated in the other java interface files dealing with Floats.  Could we put this into a separate .i file that we just include when needed, thereby keeping the driver .i's as clean as possible?